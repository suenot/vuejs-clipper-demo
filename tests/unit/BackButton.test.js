import { mount, shallowMount } from '@vue/test-utils'
import BackButton from '@/components/BackButton.vue'

describe('BackButton.vue', () => {
  it('render text', () => {
    const msg = 'Назад'
    const wrapper = shallowMount(BackButton)
    expect(wrapper.text()).toMatch(msg)
  })

  it('snapshot', () => {
    const wrapper = mount(BackButton)
    expect(wrapper.element).toMatchSnapshot()
  })
})
