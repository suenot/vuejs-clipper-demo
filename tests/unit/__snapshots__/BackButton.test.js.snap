// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackButton.vue snapshot 1`] = `
<div
  class="backButton"
>
  <img
    alt="example"
    class="icon"
    src="../../public/img/icons/corner_left.svg"
    svg-inline=""
  />
   
  <span>
    Назад
  </span>
</div>
`;
